src/media-segment-request.js

```diff
@@ -488,6 +488,16 @@ const handleSegmentBytes = ({
     return;
   }
 
+  if (typeof segment.container === 'undefined') {
+    segment.container = detectContainerForBytes(bytesAsUint8Array);
+  }
+
+  if (segment.container !== 'ts' && segment.container !== 'aac') {
+    trackInfoFn(segment, {hasAudio: true, hasVideo: false});
```

Comment:
Why are we reporting audio and not video if the container type is unknown?

Reply:
Originally this was because we were working with a bad audio container, but I guess it's possible that we could get here for bad video too. I pushed a commit that makes both `hasAudio` and `hasVideo` `false` here. Needed other changes further up the chain to support that though.